auth/src/main/AndroidManifest.xml

```diff
@@ -65,6 +65,17 @@
             android:exported="false"
             android:windowSoftInputMode="adjustResize" />
 
+        <activity
+            android:name=".ui.email.WelcomeBackEmailLinkPrompt"
+            android:label="@string/fui_title_welcome_back_password_prompt"
+            android:exported="false"
+            android:windowSoftInputMode="adjustResize" />
+
+        <activity
+            android:name=".ui.email.EmailLinkCatcherActivity"
+            android:exported="false"
+            android:windowSoftInputMode="adjustResize" />
```

Comment:
To make this really invisible, add:

```xml
android:label=""
android:theme="@style/FirebaseUI.Transparent
```

(I tested this, looks better)

Reply:
Much better! Done.